Lib/traceback.py

```diff
@@ -1732,6 +1710,26 @@ def _compute_suggestion_error(exc_value, tb, wrong_name):
     return suggestion
 
 
+def _compute_suggestion_message(exc_type, exc_value, exc_traceback) -> str | None:
```

Comment:
```suggestion
def _compute_suggestion_message(exc_type, exc_value, exc_traceback):
```

We don't add typings in the standard library usually. It's left to typeshed.